edgelet/edgelet-http-workload/src/server/mod.rs

```diff
@@ -121,7 +121,8 @@ fn get_master_encryption_key(
     key_client
         .create_key_if_not_exists(
             "iotedge_master_encryption_id",
-            aziot_key_common::CreateKeyValue::Generate { length: 32 },
+            aziot_key_common::CreateKeyValue::Generate,
+            &[aziot_key_common::KeyUsage::Encrypt],
```

Comment:
It's used for derivation in `get_derived_enc_key_handle`, so this needs to be `::Derive`